### PR TITLE
Added open AI back end/front end code but not getting any reponse.

### PR DESCRIPTION
**Pull request by samscupoftea**
Hey Josue. I've added an open Ai call to the server JS and a front end code to trigger the call upon page refresh. 

I tried my trial open AI key but it's still not responding at all. 

Perhaps you could pull this and see if you can try a different API key with it? 

I'm going to keep trying at it also later today but it's there if you want to take a look too :)

### CODE REVIEW

File: public/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+document.addEventListener('DOMContentLoaded', (event) => {
```
Comment (samscupoftea) [inline public/index.js:1]: Added an event listener to trigger when page is loaded to fetch Open Ai response.

File: server.js [hunk 2]
```diff
@@ -4,23 +4,42 @@ require('dotenv').config();
 const {configuration, OpenAI } = require ('openai');
 const app = express();
 
-// const openai = new OpenAI({
-//     apiKey: process.env.openaiAPI,
+const openai = new OpenAI({
```
Comment (samscupoftea) [inline server.js:7]: Secured Open ai Key. All you have to do is add the open AI key into .env file.

File: server.js [hunk 3]
```diff
@@ -4,23 +4,42 @@ require('dotenv').config();
...
+    apiKey: process.env.openaiAPI,
+});
```
Comment (samscupoftea) [inline server.js:9]: Commented this out but this is very similar to the active code. It includes a catch error.

**Review by samscupoftea — COMMENTED** [review]
Have a look at this code @josueJURE 👍

**Review by josueJURE — COMMENTED** [review]
I've had a look at the codebase. Looks similar to the code we wrote for other projects

**Pull request merged**